tests/conftest.py

```diff
@@ -169,43 +170,33 @@ def persisted_alice_transaction(alice_privkey, driver, alice_transaction_obj):
 
 
 @fixture
-def bicycle_asset():
+def bicycle_asset_payload():
     return {
-        'data': {
-            'bicycle': {
-                'manufacturer': 'bkfab',
-                'serial_number': 'abcd1234',
-            },
+        'bicycle': {
+            'manufacturer': 'bkfab',
+            'serial_number': 'abcd1234',
         },
-        'divisible': False,
-        'refillable': False,
-        'updatable': False,
-        'id': str(uuid4()),
     }
 
 
 @fixture
-def car_asset():
+def car_asset_payload():
     return {
-        'data': {
-            'car': {
-                'manufacturer': 'bkfab',
-                'vin': '5YJRE11B781000196',
-            },
+        'car': {
+            'manufacturer': 'bkfab',
+            'vin': '5YJRE11B781000196',
         },
-        'divisible': False,
-        'refillable': False,
-        'updatable': False,
-        'id': str(uuid4()),
     }
```

Comment:
Isn't it ok to simply use `bicycle_asset`, `car_asset`, without `_payload` since the representation of an asset is a dictionary. That is: we do not have different representations such as classes, etc.

Although it is a minor detail, it helps making the tests simpler and more lightweight.